src/api/Item.js

```diff
@@ -333,7 +334,9 @@ class Item extends Base {
             this.successCallback = successCallback;
             this.errorCallback = errorCallback;
 
-            this.validateRequest(id, permissions);
+            // if we use the default access level, we don't need permission to set the access level
+            const skipCanSetShareAccess = access === undefined;
+            this.validateRequest(id, permissions, skipCanSetShareAccess);
```

Comment:
When a Viewer or a Viewer Uploader creates a shared link using enterprise default settings, they do not need the `can_set_share_access` permission. Thus, we need to skip this permission check using a boolean